tests/mock_network/utils.rs

```diff
@@ -690,109 +715,20 @@ pub fn verify_section_invariants_for_node(node: &TestNode, elder_size: usize) {
     };
     if !all_neighbours_covered {
         panic!(
-            "{} Some neighbours aren't fully covered by the chain: {:?}",
-            node.name(),
+            "{}({:b}) Some neighbours aren't fully covered by the chain: {:?}",
+            our_name,
+            our_prefix,
             iter::once(*our_prefix)
                 .chain(neighbour_prefixes)
-                .collect::<Vec<_>>()
+                .format(", ")
         );
     }
 }
 
-pub fn verify_section_invariants_for_nodes(nodes: &[TestNode], elder_size: usize) {
-    for node in nodes.iter() {
-        verify_section_invariants_for_node(node, elder_size);
-    }
-}
-
-pub fn verify_section_invariants_between_nodes(nodes: &[TestNode]) {
```

Comment:
do we still need to verify the consistency among the nodes?

Reply:
Well that's one of the main points of this PR - this function was incorrect because we should never expect that all nodes have the same view of the network at the same time. That's not how the real network works and so we should not assume that in the tests either.